cmdLine/src/main/scala/topgun/cmdline/AsmRecord.scala

```diff
@@ -0,0 +1,86 @@
+package topgun.cmdline
+
+import java.io.{IOException, InputStream, RandomAccessFile}
+import java.nio.file.{Files, Paths}
+import java.util.HashMap
+
+import org.objectweb.asm.ClassReader
+import org.objectweb.asm.tree.ClassNode
+
+import scala.collection.mutable
+
+
+object AsmRecord {
+  private var map: HashMap[String, ClassInfo] = _
+  private var buildDirectory: String = ""
+
+  def peek(): Unit = {
+    println(map)
+  }
+
+  private def getPackageName(path: String): String = {
+    var packageName: String =
+      path.substring(buildDirectory.length + 1).replace('/', '.')
+    packageName = packageName.substring(0, packageName.length - 6)
+    packageName
+  }
+
+  private def build(path: String): Unit = {
```

Comment:
please use a classloader to access the resource

Reply:
okay made that change.